Added JSON data test for randomisers (#58)
ieahleen/habitican-randomizer: json_files/jsonFiles.md

## Description of JSON files
### This folder contains the json schema for the responses of the API calls made by habitican-randomiser

> * Equip [Random Pet](equipPet.json) : This JSON file contains the schema for equpping a random pet from habitica. 
> 
> The API call it responds to is: ```https://habitica.com/api/v3/user/equip/pet/{pet_name}```

Reviewer: ```suggestion
* Equip [Random Pet](equipPet.json) : This JSON file contains the schema for equpping a random pet from habitica. 

The API call it responds to is: `https://habitica.com/api/v3/user/equip/pet/{pet_name}`
```

> * Equip [Random Background](equipBGcolor.json) :  This JSON file contains the schema for equipping a random backgroundd from habitica
> 
> The API call it responds to is : ``` https://habitica.com/api/v3/user/unlock?path=background.{color} ```

Reviewer: ```suggestion
* Equip [Random Background](equipBGcolor.json) :  This JSON file contains the schema for equipping a random backgroundd from habitica

The API call it responds to is : `https://habitica.com/api/v3/user/unlock?path=background.{color}`
```

> * [Buy New Equpiment](buyNewEquip.json) : This JSON file contains the schema for buying a new quipment from the market in habitica. 
> 
> The API call it responds to is: ```https://habitica.com/api/v3/user/buy-gear/{itemToPurchase.key}```

Reviewer: ```suggestion
* [Buy New Equpiment](buyNewEquip.json) : This JSON file contains the schema for buying a new quipment from the market in habitica. 

The API call it responds to is: `https://habitica.com/api/v3/user/buy-gear/{itemToPurchase.key}`
```

> * Equip [Random Costume](costumeRandom.json) :  This JSON file contains the schema for equipping a random costume from habitica
> 
> The API call it responds to is : ``` https://habitica.com/api/v3/user/equip/costume/{costume_name} ``` 

Reviewer: ```suggestion
* Equip [Random Costume](costumeRandom.json) :  This JSON file contains the schema for equipping a random piece of equipment ascostume from habitica

The API call it responds to is : `https://habitica.com/api/v3/user/equip/costume/{costume_name}` 
```

> [Random Battle Gear](battlegearRndm.json) : This JSON file contains the schema for equipping a random battle gear from habitica
> 
> The API call it responds to is : ``` https://habitica.com/api/v3/user/equip/equipped/{item} ```

Reviewer: ```suggestion
[Random Battle Gear](battlegearRndm.json) : This JSON file contains the schema for equipping a randompiece of equipment as battle gear from habitica

The API call it responds to is : `https://habitica.com/api/v3/user/equip/equipped/{item}`
```

## Description of JSON files
### This folder contains the json schema for the responses of the API calls made by habitican-randomiser

> * Equip [Random Pet](equipPet.json) : This JSON file contains the schema for equpping a random pet from habitica. 
> 
> The API call it responds to is: `https://habitica.com/api/v3/user/equip/pet/{pet_name}`

Reviewer: ```suggestion
* Equip [Random Pet](equipPet.json) is the response to `https://habitica.com/api/v3/user/equip/pet/{pet_name}`

This is the API endpoint to equip a new pet
```

> * Equip [Random Background](equipBGcolor.json) :  This JSON file contains the schema for equipping a random backgroundd from habitica
> 
> The API call it responds to is : ` https://habitica.com/api/v3/user/unlock?path=background.{color} `

Reviewer: ```suggestion
* Equip [Random Background](equipBGcolor.json) is the response to `https://habitica.com/api/v3/user/unlock?path=background.{color}`

This is the call to equip a background.
```

> * [Buy New Equpiment](buyNewEquip.json) : This JSON file contains the schema for buying a new quipment from the market in habitica. 
> 
> The API call it responds to is: `https://habitica.com/api/v3/user/buy-gear/{itemToPurchase.key}`

Reviewer: ```suggestion
* [Buy New Equpiment](buyNewEquip.json) is the response to `https://habitica.com/api/v3/user/buy-gear/{itemToPurchase.key}`

This is the API endpoint to buy a piece of gear.
```

> * Equip [Random Costume](costumeRandom.json) :  This JSON file contains the schema for equipping a random piece of equipment as costume from habitica
> 
> The API call it responds to is : ` https://habitica.com/api/v3/user/equip/costume/{costume_name} `

Reviewer: ```suggestion
* Equip [Random Costume](costumeRandom.json) is the response to `https://habitica.com/api/v3/user/equip/costume/{costume_name}`

It's the API endpoint to equip a piece of gear as costume
```

> * [Random Battle Gear](battlegearRndm.json) : This JSON file contains the schema for equipping a random piece of equipment as battle gear from habitica
> 
> The API call it responds to is : ` https://habitica.com/api/v3/user/equip/equipped/{item} `

Reviewer: ```suggestion
* [Random Battle Gear](battlegearRndm.json) is the response to `https://habitica.com/api/v3/user/equip/equipped/{item}`

Same as above but for battle gear
```

> * [User](user.json): This JSON file contains the details for a user in habitica
> 
> The API call it responds to is : ` https://habitica.com/api/v3/user `

Reviewer: ```suggestion
* [User](user.json) is the response to `https://habitica.com/api/v3/user`

This is the API call to get all the user data
```

> * [Content](content.json) : This JSON file contains the details of all the content used by a user in habitica
> 
> The API call it responds to is : ` https://habitica.com/api/v3/content?language=en `

Reviewer: ```suggestion
* [Content](content.json) is the response to `https://habitica.com/api/v3/content?language=en`

It's all available Habitica content.
```
